### PR TITLE
New package: py-bioepic (and it's dependencies) (SL-609)

### PR DESCRIPTION
**Pull request by rmsds**
* adding py-bioepic: https://github.com/biocore-ntnu/epic
* dependencies:
  * updates (of any not yet installed notable packages): py-pandas / jellyfish / py-cryptography
  * imported from upstream: py-docopt
  * new packages: py-pyopenssl / py-coveralls

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-bioepic/package.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    homepage = "https://github.com/biocore-ntnu/epic"
+    git      = "https://github.com/biocore-ntnu/epic"
+    url      = "https://pypi.io/packages/source/b/bioepic/bioepic-0.2.12.tar.gz"
+
+    version('develop', branch='master')
+    version('0.2.12', 'aa97bc47449b8491121d447161ad5542')
+
+    depends_on('jellyfish@2:')
+    depends_on('py-coverage', type=('build'))
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-bioepic/package.py]: Just a sidenote: `('build')` is an expression which evaluates to `'build'`, while `('build',)` is a tuple. **TL;DR** I think the parenthesis are unnecessary. :smile:

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
I'll check the failure in Travis tomorrow, will push a commit here to improve error reporting on Travis.

**Comment by alalazo** [comment]
This branch is kept open to check why we have a failure on Travis (it is claimed that some package here is part of the DAG of packages we use for build tests).

**Comment by rmsds** [comment]
The issue this was being used to debug is fixed by https://github.com/epfl-scitas/spack/pull/216/commits/e049dd980d8196fa70d59a142526ff51af92171a. Closing.

**Pull request closed without merging**